src/localplayer.cpp

```diff
@@ -298,11 +297,24 @@ void LocalPlayer::move(f32 dtime, Environment *env, f32 pos_max_d,
 			new_sneak_node = p;
 		}
 
-		bool sneak_node_found = (min_distance_f < 100000.0*BS*0.9);
+		bool sneak_node_found = (min_distance_f < 100000.0 * BS * 0.9);
 
 		m_sneak_node = new_sneak_node;
 		m_sneak_node_exists = sneak_node_found;
 
+		if (sneak_node_found) {
+			f32 cb_max = 0;
```

Comment:
Hmm, nevermind this, I think we could optimize it later on.  The code here gets executed only when moving to a new node while sneaking.